x/tokenomics/keeper/keeper_settle_pending_claims_test.go

```diff
@@ -303,11 +303,13 @@ func (s *TestSuite) TestSettlePendingClaims_ClaimExpired_ProofRequiredAndNotProv
 	require.Equal(t, uint64(0), slashedSupplier.UnstakeSessionEndHeight)
 
 	events := sdkCtx.EventManager().Events()
-	require.Len(t, events, 10) // asserting on the length of events so the developer must consciously update it upon changes
+	// TODO_IN_THIS_COMMIT: figure out why events went from 10 to 6.
```

Comment:
@okdas Can you tell why this didn't trigger?

@bryanchriswhite Making sure you didn't forget this.

Reply:
Hehe - thanks! :sweat_smile: 

I'm at a loss at the moment for an explanation to this; however, all unit, integration, and E2E tests are passing.

May I propose the following for now?
```suggestion
	// TODO_MAINNET(@bryanchriswhite)/TODO_INVESTIGATE: figure out why events went from 10 to 6 in PR#889.
	// (https://github.com/pokt-network/poktroll/pull/889)
	// Seem to be missing a message, trasnfer, coin_spent, and coin_received.
```